index.js

```diff
@@ -7,7 +7,20 @@ var Q = require('q');
 var cordovaLib = require('cordova-lib').cordova;
 var cordova = cordovaLib.raw;
 
-module.exports = function (rm) {
+module.exports = function (options) {
+	options = options || {};
+	var rm, version;
```

Comment:
No need for backwards compatibility, will release as breaking.

```js
options = options || {};
```

Remove the entire if block and use `options.reAdd` further down the road.